packages/gcf-utils/test/logging-octokit-plugin.ts

```diff
@@ -357,18 +357,4 @@ describe('Logging-Octokit-Plugin', () => {
         assert.deepEqual(actual, expected);
       });
   });
-
-  it('does not log information for unknown actions', () => {
```

Comment:
I'm not quite sure what the intent of this test was @chingor13 @azizsonawalla? but I couldn't seem to get it working on the latest `@octokit/rest`.

Reply:
I believe we had to explicitly maintain a map of octokit functions to the data that we log. I think this test uses an untracked endpoint and thus should not log any actions.

Reply:
Yep, @chingor13 is right. We only wanted to log certain calls to Octokit that bots make (i.e. calls to Octokit that actually make a change on GitHub like updating an issue, rather than just 'get' information). So this test checks that a non-tracked action does not log any information.

Reply:
added test back.